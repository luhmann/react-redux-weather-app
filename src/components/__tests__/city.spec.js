import React from 'react'
import { shallow } from 'enzyme'

import City from '../city'

describe('Component: City', () => {
  it('should render as expected when not loading', () => {
    const subject = shallow(
      <City
        degrees={20}
        iconUrl="http://example.org/test.png"
        name="Berlin"
        isLoading={false}
      />
    )

    expect(subject).toMatchSnapshot()
  })

  it('should render as expected when loading', () => {
    const subject = shallow(<City isLoading name="Berlin" />)

    expect(subject).toMatchSnapshot()
  })
})
