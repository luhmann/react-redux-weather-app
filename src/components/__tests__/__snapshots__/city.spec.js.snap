// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: City should render as expected when loading 1`] = `
<div
  className="city"
>
  Loading data for 
  Berlin
</div>
`;

exports[`Component: City should render as expected when not loading 1`] = `
<div
  className="city"
>
  <div
    className="city__name"
  >
    Berlin
  </div>
  <div
    className="city__info"
  >
    <img
      src="http://example.org/test.png"
    />
    20
    Â°C
  </div>
</div>
`;
